### PR TITLE
add required to quick filter bar

### PR DESCRIPTION
**Pull request by micaelpc**

### CODE REVIEW

**Review by Sagi-o — APPROVED** [review]

**Comment by doroncy** [comment]
how does it look when its required ?

**Comment by micaelpc** [comment]
@doroncy same - but with * above and not allow for de-select item (on single select)

File: projects/ui-framework/src/lib/search/quick-filter/quick-filter.component.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
                   [label]="quickFilterConfig.label"
                   [placeholder]="quickFilterConfig.placeholder || null"
                   [options]="quickFilterConfig.options"
+                  [required]="quickFilterConfig.required"
```
Comment (galloween) [inline projects/ui-framework/src/lib/search/quick-filter/quick-filter.component.html:12]: replace the `[required]="false"` below

Comment (micaelpc) [reply projects/ui-framework/src/lib/search/quick-filter/quick-filter.component.html:12]: DONE

**Review by doroncy — APPROVED** [review]

**Pull request merged**